### ISSUE TITLE
MkDocs improvements

### ISSUE THREAD

**Issue by vincentarelbundock** [etiennebacher/altdoc#157]
Things I notice while converting `marginaleffects` to `altdoc`:

+ [ ] repo_url should be the Github URL
+ [ ] `- extra_css` doesn't work because `write_yaml()` converts to string rather than keep as list.

**Comment by etiennebacher** [comment]
The second point was already addressed in #156 I think

**Comment by vincentarelbundock** [comment]
@etiennebacher Would you happen to know what the problem is with this deployment? Github is setup to deploy from `main/docs` (*not* from the alternate branch), but I can't reach the main website without inserting `index.html` explicitly in the URL. And even then, all the links are broken.

https://github.com/vincentarelbundock/marginaleffects/tree/main/docs
https://vincentarelbundock.github.io/marginaleffects/
https://vincentarelbundock.github.io/marginaleffects/index.html

**Comment by etiennebacher** [comment]
https://vincentarelbundock.github.io/marginaleffects/ and https://vincentarelbundock.github.io/marginaleffects/index.html both lead to the homepage for me.

Links to vignettes / man show the rendered vignette but not as an mkdocs page: 

![image](https://github.com/etiennebacher/altdoc/assets/52219252/5efd4058-de46-47e2-bd6b-70ae423eaa77)

I'm taking a look

**Comment by vincentarelbundock** [comment]
Thanks, looks like it's working now, but I had to add `/` at the end of every link. It might be worth it to check and add this automatically in `R/settings_mkdocs.R`

**Comment by etiennebacher** [comment]
Yup, sounds good, the `/` is also required on the r-polars website

**Comment by etiennebacher** [comment]
Just a sidenote but I love the mkdocs website for marginaleffects, I prefer it to the quarto website

**Comment by etiennebacher** [comment]
I don't think we should add the option to remove the footer, even commented out:

![image](https://github.com/etiennebacher/altdoc/assets/52219252/830dd879-5f86-48fd-a234-081754ab8122)
